t/RpmSpecMaker.t

```diff
@@ -4,51 +4,75 @@ use JSON::Fast;
 
 use RpmSpecMaker;
 
-subtest {
-    dies-ok { generate-spec('No valid json') }, "Invalid json dies";
-
-    my $valid-json = q:to/END/;
-    {
-       "authors" : [
-          "pnu",
-          "wbiker"
-       ],
-       "build-depends" : [],
-       "depends" : [],
-       "description" : "This is a generic module for interactive prompting from the console.",
-       "license" : "Artistic-2.0",
-       "name" : "IO::Prompt",
-       "perl" : "6.*",
-       "provides" : {
-          "IO::Prompt" : "lib/IO/Prompt.pm"
-       },
-       "resources" : [],
-       "source-url" : "http://www.cpan.org/authors/id/W/WB/WBIKER/Perl6/IO-Prompt-0.0.2.tar.gz",
-       "tags" : [],
-       "test-depends" : [
-          "Test"
-       ],
-       "version" : "0.0.2"
-    }
-    END
+rm-directories("packages".IO);
```

Comment:
That looks quite dangerous. Depending on where you start that test from, it will delete some directory named "packages". It'd be better to make the target directory dynamic and then use a temporary directory for tests.

Reply:
You are right. Refactored. Now the directory is injected and the File::Temp module is used in tests.